snakebite/client.py

```diff
@@ -383,9 +383,8 @@ def _handle_rename(self, path, node, dst):
         if not dst.startswith("/"):
             dst = self._join_user_path(dst)
 
-        # Strip the last / if there is one. Hadoop doesn't like this
-        if dst.endswith("/"):
-            dst = dst[:-1]
+        dst = os.path.normpath(re.sub('/+', '/', dst))
```

Comment:
Should `re.sub('/+', '/', dst)` go to helper method to reduce duplication - it is still one line, but is used at least 4 times.